code/datums/mood_events/needs_events.dm

```diff
@@ -20,6 +20,14 @@
 	mood_change = -10
 
 //charge
+/datum/mood_event/supercharged
+	description = "<span class='boldwarning'>I can't possibly keep all this power inside, I need to release some quick!</span>\n"

```

Comment:
ethereal version of i need to COOOOOOOOM

Reply:
😏